advisor/build.gradle.kts

```diff
@@ -23,10 +23,10 @@ plugins {
 }
 
 dependencies {
-    api(project(":model"))
```

Comment:
I've done a `grep` and the output makes me wonder if (at least some of) the following should also be changed:

```
buildSrc/src/main/kotlin/ort-kotlin-conventions.gradle.kts:                implementation(project(":utils:test-utils"))
buildSrc/src/main/kotlin/ort-kotlin-conventions.gradle.kts:    "detektPlugins"(project(":detekt-rules"))
detekt-rules/build.gradle.kts:        substitute(project(":utils:test-utils"))
detekt-rules/build.gradle.kts:            .using(project(":utils:common-utils"))
plugins/package-managers/gradle/src/funTest/assets/projects/synthetic/multi-kotlin-project/cli/build.gradle.kts:    implementation(project(":core"))
settings.gradle.kts:project(":clients:clearly-defined").name = "clearly-defined-client"
settings.gradle.kts:project(":clients:fossid-webapp").name = "fossid-webapp-client"
settings.gradle.kts:project(":clients:github-graphql").name = "github-graphql-client"
settings.gradle.kts:project(":clients:nexus-iq").name = "nexus-iq-client"
settings.gradle.kts:project(":clients:oss-index").name = "oss-index-client"
settings.gradle.kts:project(":clients:osv").name = "osv-client"
settings.gradle.kts:project(":clients:scanoss").name = "scanoss-client"
settings.gradle.kts:project(":clients:vulnerable-code").name = "vulnerable-code-client"
settings.gradle.kts:project(":utils:common").name = "common-utils"
settings.gradle.kts:project(":utils:ort").name = "ort-utils"
settings.gradle.kts:project(":utils:scripting").name = "scripting-utils"
settings.gradle.kts:project(":utils:spdx").name = "spdx-utils"
settings.gradle.kts:project(":utils:test").name = "test-utils"
settings.gradle.kts:        project(":$projectPath").name = when(parts[2]) {
```